src/run/runner/wall_time/perf/shared.rs

```diff
@@ -0,0 +1,22 @@
+// !!!!!!!!!!!!!!!!!!!!!!!!
+// !! DO NOT TOUCH BELOW !!
+// !!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
@not-matthias same here for the decoration, adding a warning below would be good enough